src/clusto/services/http.py

```diff
@@ -205,6 +205,41 @@ def show(self, request, prefetch_attrs=None):
 
         return dumps(request, result)
 
+    def rename(self, request):
+        '''
+        Renames an object and returns the new object.
+
+        Example: /server/exampleserver/rename?object=/server/newname
+        '''
+        device_type, device_name = request.params['object'].strip('/').split('/')
```

Comment:
I'm surprised there isn't a method for parsing clusto paths, since every bloody function does it.